suite-native/app/ios/TrezorSuite/Images.xcassets/SplashScreen.imageset/Contents.json

```diff
@@ -1,17 +1,17 @@
 {
     "images": [
         {
-            "filename": "splash_icon.png",
+            "filename": "splash_icon.svg",
             "idiom": "universal",
             "scale": "1x"
         },
         {
-            "filename": "splash_icon@2x.png",
+            "filename": "splash_icon 1.svg",
```

Comment:
I not sure if empty spaces in filenames are good idea

Reply:
I let only one file definition because the file is svg so it should scale. 6eb52e06277547303899f873a87bcd4f9fcddf50